scripts/ci/install-istio.sh

```diff
@@ -26,14 +28,15 @@ install_istio() {
                 exit 1
         fi
 
-        chmod a+x $ISTIO_PKG
-        sudo cp $ISTIO_PKG/bin/istioctl $TARGET_DIR/.
-        sudo mv $ISTIO_PKG $TARGET_DIR/.
+        sudo cp -R $ISTIO_PKG $TARGET_DIR
```

Comment:
don't install all istio package in /usr/bin .... we don't want to put files where they not belonging .... install it properly

Reply:
That's precisely the point of this PR :-) Install Istio in /opt and only put the executable in /usr/bin